scss/_base/_forms.scss

```diff
@@ -156,19 +156,97 @@ select {
   }
 }
 
-// Checkboxes and radio boxes. Must be enclosed in `.option-group` class.
+// Checkboxes and radio boxes.
 //
 // Styleguide Forms - Option Fields
-.option-group, .form-item.form-type-checkbox {
-  margin-bottom: $base-spacing / 2;
+.option {
+  // Hat tip to Mark Otto's [WTF, Forms](http://wtfforms.com)
+  // for markup inspiration and Open Iconic for check and radio icons.
+  //
+  // https://useiconic.com/open/
+  // Open Iconic released under MIT License.
+
+  position: relative;
+  display: inline-block;
+  height: auto;
+  padding-left: $base-spacing;
+  margin: ($base-spacing / 4) 0;
+  cursor: pointer;
+
+  .option__indicator {
+    position: absolute;
+    top: 4px;
+    left: 0;
+    display: block;
+    width: 16px;
+    height: 16px;
+    color: $light-gray;
+    background-color: #eee;
+    background-size: 50% 50%;
+    background-position: center center;
+    background-repeat: no-repeat;
+    user-select: none;
+  }
+
+  input {
+    position: absolute;
+    opacity: 0;
+    z-index: -1; // Put the input behind the label so it doesn't overlay text
+  }
```

Comment:
Maybe use the `@include visually_hidden()` mixin added to Neue to hide this? It doesn't rely on opacity so it also supports IE8 ;)